src/Entity/FooterLinkSocial.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Drupal\oe_corporate_blocks\Entity;
+
+/**
+ * Defines the footer link social configuration entity.
+ *
+ * @ConfigEntityType(
+ *   id = "footer_link_social",
+ *   label = @Translation("Footer Link Social"),
+ *   handlers = {
+ *     "view_builder" = "Drupal\Core\Entity\EntityViewBuilder",
+ *     "list_builder" = "Drupal\oe_corporate_blocks\FooterLinkSocialListBuilder",
+ *     "route_provider" = {
+ *       "html" = "Drupal\Core\Entity\Routing\AdminHtmlRouteProvider",
+ *     },
+ *     "form" = {
+ *       "add" = "Drupal\oe_corporate_blocks\Form\FooterLinkSocialForm",
+ *       "edit" = "Drupal\oe_corporate_blocks\Form\FooterLinkSocialForm",
+ *       "delete" = "Drupal\oe_corporate_blocks\Form\FooterLinkGeneralDeleteForm"
+ *     }
+ *   },
+ *   config_prefix = "footer_link.social",
+ *   admin_permission = "administer footer link social",
+ *   entity_keys = {
+ *     "id" = "id",
+ *     "label" = "label",
+ *     "social_network" = "social_network",
+ *     "uuid" = "uuid",
+ *     "weight" = "weight",
+ *   },
+ *   config_export = {
+ *     "id",
+ *     "label",
+ *     "social_network",
+ *     "url",
+ *     "weight"
+ *   },
+ *   links = {
+ *     "add-form" = "/admin/config/footer-link-social/add",
+ *     "edit-form" = "/admin/config/footer-link-social/{footer_link_social}/edit",
+ *     "delete-form" = "/admin/config/footer-link-social/{footer_link_social}/delete",
+ *     "collection" = "/admin/config/footer-link-social"
```

Comment:
We move this one under structure, we did the same with breadcrumbs on EWCMS. There we used the following:
```
 *     "add-form" = "/admin/structure/breadcrumb_root_item/add",
 *     "edit-form" = "/admin/structure/breadcrumb_root_item/{breadcrumb_root_item}/edit",
 *     "delete-form" = "/admin/structure/breadcrumb_root_item/{breadcrumb_root_item}/delete",
 *     "collection" = "/admin/structure/breadcrumb_root_item"
```

So let's do as we did there and use the entity ID `footer_link_social`:

```
 *     "add-form" = "/admin/structure/footer_link_social/add",
 *     "edit-form" = "/admin/structure/footer_link_social/{footer_link_social}/edit",
 *     "delete-form" = "/admin/structure/footer_link_social/{footer_link_social}/delete",
 *     "collection" = "/admin/structure/footer_link_social"
```

Reply:
Agreed to keep it under `/config` but to still use the entity ID in the path.